playground/index.js

```diff
@@ -0,0 +1,16 @@
+const linear = require('./node_modules/linear-debounce/dist/linear');
```

Comment:
Either create this script in the npm "prepare" script:
```sh
mkdir -p playground/node_modules
ln -s ../../playground/node_modules/linear-debounce
```

Or add line:
```
!playground/node_modules
```
to you your `.gitignore` file

Reply:
Good call.